### PR TITLE
Update delete portfolio modal

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://projects.engineering.redhat.com/browse/SSP-1693

### Before
screenshot-ci cloud redhat com-2020 07 28-11_49_24

### After
screenshot-ci foo redhat com_1337-2020 07 28-11_46_55

### CODE REVIEW

File: src/smart-components/portfolio/remove-portfolio-modal.js [hunk 1]
```diff
@@ -48,7 +49,21 @@ const RemovePortfolioModal = ({ viewState }) => {
     <UnauthorizedRedirect />
   ) : (
     <Modal
-      title={formatMessage(portfolioMessages.portfolioRemoveTitle)}
+      aria-label="Delete portfolio modal"
```
Comment (rvsia) [inline src/smart-components/portfolio/remove-portfolio-modal.js:52]: This should be translatable too.

File: src/smart-components/portfolio/remove-portfolio-modal.js [hunk 2]
```diff
@@ -48,7 +49,21 @@ const RemovePortfolioModal = ({ viewState }) => {
...
+            <SplitItem>
+              <ExclamationTriangleIcon size="lg" fill="#f0ab00" />
+            </SplitItem>
+            <SplitItem>
+              <Text component="h1" size="2xl">
+                {formatMessage(portfolioMessages.portfolioRemoveTitle)}
+              </Text>
+            </SplitItem>
+          </Split>
+        </TextContent>
```
Comment (rvsia) [inline src/smart-components/portfolio/remove-portfolio-modal.js:65]: This markup seems too complicated.

You can check how it's done in Sources: https://github.com/RedHatInsights/sources-ui/blob/master/src/components/AddApplication/RemoveAppModal.js#L53

File: src/smart-components/portfolio/remove-portfolio-modal.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useHistory } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
-import { WarningTriangleIcon } from '@patternfly/react-icons';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
```
Comment (rvsia) [inline src/smart-components/portfolio/remove-portfolio-modal.js:5]: Don't you use transform-imports, do you?

Comment (Hyperkid123) [reply src/smart-components/portfolio/remove-portfolio-modal.js:5]: I do I just got used to full imports :D

File: src/smart-components/portfolio/remove-portfolio-modal.js (hunk 2 above)
Comment (Hyperkid123) [reply src/smart-components/portfolio/remove-portfolio-modal.js:65]: Its PF layout. I think I will stick with it.

**Pull request merged**